test.disabled/ejabberd_tests/tests/muc_SUITE.erl

```diff
@@ -2599,16 +2604,98 @@ one2one_chat_to_muc(Config) ->
         escalus_assert:has_no_stanzas(Eve)
     end).
 
+registration_query_service(Config) ->
+    escalus:story(Config, [{alice, 1}, {bob, 1}], fun(_Alice,  Bob) ->
+        escalus:send(Bob, escalus_stanza:to(escalus_stanza:iq_get(<<"jabber:iq:register">>, []), ?MUC_HOST)),
+   	    Resp = escalus:wait_for_stanza(Bob),
+        escalus:assert(is_iq_result, Resp),
+        % assert the response instructs to submit a nick
+        Field = exml_query:path(Resp, [{element, <<"query">>}, {element, <<"x">>}, {element, <<"field">>}]),
+        ?assert_equal(<<"text-single">>, exml_query:path(Field, [{attr, <<"type">>}])),
+        ?assert_equal(<<"nick">>, exml_query:path(Field, [{attr, <<"var">>}])),
+        ok
+    end).
+
+query_register_nick(Nick) ->
+    [
+        #xmlel{name = <<"x">>,
+               attrs = [{<<"xmlns">>, <<"jabber:x:data">>}, {<<"type">>, <<"submit">>}],
+               children = [
+                   #xmlel{name = <<"field">>,
+                          attrs = [{<<"type">>, <<"text-single">>}, {<<"var">>, <<"nick">>}],
+                          children = [
+                              #xmlel{name = <<"value">>,
+                                     children = [#xmlcdata{content = Nick}]}]}
+               ]}
+    ].
+
+
+stanza_register_nick(Nick, To) ->
+    Req = escalus_stanza:iq_set(<<"jabber:iq:register">>,
+                                query_register_nick(Nick)),
+    case To of
+        ?MUC_HOST ->
+            escalus_stanza:to(Req, To);
+        Room ->
```

Comment:
Are `Room` and `{Room, Nick}` used anywhere?

Reply:
From what I read in the code, this function provides a way to send stanza to a room, with or without nick, so yes, they are. There is no test for it because there is no implementation for it, there is only a stub test and it is commented out. So no, they are not.